jobtracker.go

```diff
@@ -422,7 +424,7 @@ func (jt *jobTracker) FetchJobDetails(id string, host string) (*jobDetail, error
 	var url string
 	appID, jobID := hadoopIDs(id)
 	if host == jt.rm {
-		url = fmt.Sprintf("%s/proxy/%s/ws/v1/mapreduce/jobs", host, appID)
+		url = fmt.Sprintf("%s/proxy/%s/ws/v1/mapreduce/jobs", jt.ps, appID)
```

Comment:
Unfortunately yes. The YARN ProxyServer only serves the /proxy/ namespace, and all the other API calls need to go direct to the ResourceManager. When the yarn.web-proxy.address option is set, the RM stops responding on the /proxy/ endpoints, so given the way Timberlake builds the URLs, it does need to know about the separation. 

However, now I've looked a bit more closely, the RM returns the correct URL in the trackingUrl field of the /ws/v1/cluster/apps/ response. Using the URL from there instead of generating it from the appID would be cleaner.